manifest.yml

```diff
@@ -0,0 +1,12 @@
+---
+applications:
+ - name: paas-usage-events-collector
+   memory: 128M
+   disk_quota: 100M
+   instances: 1
+   buildpack: go_buildpack
+   no-route: true
```

Comment:
I think this is causing deploys in my dev environment to fail. I get the following error:

```
Could not unmap route - Route paas-usage-events-collector-new not found
```

It looks as though `blue-green-deploy` relies on having a predictable route based off the application name?

Reply:
For an app without a route, there shouldn't be a need to do a blue-green deploy at all should there?

Reply:
You are right, I changed it to cf push instead.